crawl-ref/source/dat/des/portals/bailey.des

```diff
@@ -1257,3 +1257,57 @@ ccWWc+c+c!!!!!!cc
    cc...cc
     ccccc
 ENDMAP
+
+# A short but intense spriggan/ogre bailey
+# Nine items, with a chance for some delicious dragon scales
+NAME:   bailey_polearm_enchanted_keep_hellmonk
```

Comment:
The first time I played through this I first encountered ogres then the ogre spriggan mix, then the inner baddies. That felt like a good progression, but with the way the map rolls the second play through I rolled spriggans first, which made the following ogres feel kind of silly. Maybe de-randomize the placement a little bit to get better pacing.